src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -196,7 +200,14 @@ public void start() {
 
         // Handles operations
         eb.consumer(address, (Handler<Message<JsonObject>>) event -> {
-            String operation = event.body().getString(OPERATION);
+            final JsonObject body = event.body();
+            if (null == body) {
```

Comment:
Why `null == body` and not `body == null`? The latter is better readable like 'natural language' - and you use this style elsewhere